examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java

```diff
@@ -55,14 +52,47 @@ public static void addOPCOutput(LX lx) throws Exception {
     lx.engine.addOutput(new OPCOutput(lx, "localhost", 7890));
   }
 
+
+  public static void addTenereOutput(LX lx, String ip) throws Exception {
+    lx.engine.addOutput(
+            new LXDatagramOutput(lx).addDatagram(
+                    new TenereDatagram(lx, LXOutput.fixtureToIndices(lx.model), (byte) 0x00).setAddress(ip).setPort(1337))
+                    .addDatagram(
+                    new TenereDatagram(lx, LXOutput.fixtureToIndices(lx.model), (byte) 0x04).setAddress(ip).setPort(1337))
+    );
+  }
+
+
+  public static void addDatagramOPCOutput(LX lx, String ip_addr) throws Exception {
+    lx.engine.addOutput(
+            new LXDatagramOutput(lx).addDatagram(
+                    new OPCDatagram(lx.model)
+                            .setAddress(ip_addr)
+                            .setPort(7890)
+
+            )
+    );
+  }
+
   public static void main(String[] args) {
     try {
       LXModel model = buildModel();
       LX lx = new LX(model);
 
+
+      addTenereOutput(lx, "10.200.1.64");
+      addTenereOutput(lx, "10.200.1.65");
+
+      // target some OPC servers
+      String[] controller_ips = { "10.200.1.146", "10.200.1.195", "10.200.1.197"};
```

Comment:
Minor naming convention thing, but should keep the style conventions standard in here. lower_caps_underscore shouldn't be used here, rather should be camelCase.

I'm also not sure these specific IP addresses and Tenere-specific output classes ought to go in the base project, happy to let this slide but let's try to keep project-specific stuff to a minimum in here.